package.json

```diff
@@ -60,18 +61,18 @@
     "parser": "babel-eslint"
   },
   "scripts": {
-    "lint": "node node_modules/standard/bin/cmd.js src/**/*.js",
```

Comment:
Will it prioritize `node_modules` binaries over global ones? K now I'm googling it.